js1.js

```diff
@@ -0,0 +1,82 @@
+let todo_button = document.getElementById('todo_button');
+let todo_box = document.getElementById('todo_box');
+let todo_space = document.getElementById('todo_space');
+let lenght = document.getElementById('number');
+
+count();
+function add_todo(){
+  if(todo_box.value!=""){ //입력 있을 때만 추가

```

Comment:
```suggestion
  if(todo_box.value == "" ) {
    return;
  }
```

입력 없는 경우를 미리 return 해버리면 코드 들여쓰는 부분이 없고 예외 처리를 미리 해두기 때문에 깔끔해질 것 같아. 이런 패턴을 early return 이라고 불러.